pkg/cmd/kubernetes_exec.go

```diff
@@ -193,12 +263,21 @@ func getDefaultKubeconfig() string {
 	return kubeConfigPath
 }
 
-func tryDownloadHelm(userPath, clientArch, clientOS string) (string, error) {
-	helmBinaryPath := path.Join(path.Join(userPath, "bin"), "helm")
+func tryDownloadHelm(userPath, clientArch, clientOS string, helm3 bool) (string, error) {
+	helmVal := "helm"
+	if helm3 {
+		helmVal = "helm3"
+	}
+
+	helmBinaryPath := path.Join(path.Join(userPath, "bin"), helmVal)
```

Comment:
I think homebrew installs helm3 as helm, or helm@3? So wont this not work in that case?

Reply:
this creates:

.k3sup/bin/helm3/helm